generation/diff_tool.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+#run this script from google-cloud-java directory
+# google-cloud-java$ ./generation/diff-tool.sh
```

Comment:
Clarify the output of this script. I don't want to check 100 files one by one. How can we avoid that?

Reply:
please see [this](https://github.com/googleapis/google-cloud-java/pull/8251) for the latest changes.

Reply:
Do you want to close this 8427 in favor of 8251?